galite-tests/src/test/kotlin/org/kopi/galite/tests/ui/vaadin/report/ReportTests.kt

```diff
@@ -80,6 +83,105 @@ class ReportTests: GaliteVUITestBase() {
     }
   }
 
+  @Test
+  fun `test export CSV`() {
+    // Login
+    login()
+
+    // Opens a form that contain a report command
+    formWithReport.open()
+
+    // Trigger the report command
+    formWithReport.report.triggerCommand()
+
+    // Check that the report is displayed
+    _expectOne<DReport>()
+    simpleReport.csv.triggerCommand()
+
+    _expectOne<DownloaderDialog>()
+    val downloadDialog = _get<DownloaderDialog>()
+
+    assertTrue(downloadDialog.isOpened)
+    val anchor = downloadDialog._get<Anchor> {  }
+
+    assertTrue(anchor.href.contains(simpleReport.title))
```

Comment:
you can assert also on` div with class='download-file-title' `end with CSV (add it to pdf, xls and xlsx tests)